bench/llvm/optimized/Target.ll

```diff
@@ -177,29 +177,35 @@ _ZN4llvmeqENS_9StringRefES0_.exit.i.i78:          ; preds = %10
 _ZN4llvmeqENS_9StringRefES0_.exit.i.i86:          ; preds = %10
   %bcmp.i.i.i87 = call i32 @bcmp(ptr noundef nonnull dereferenceable(9) %14, ptr noundef nonnull dereferenceable(9) @.str.10, i64 9)
   %28 = icmp eq i32 %bcmp.i.i.i87, 0
-  br i1 %28, label %_ZN4llvm12StringSwitchINS_5MachO12PlatformTypeES2_E4CaseENS_13StringLiteralES2_.exit105, label %_ZNK4llvm9StringRef11starts_withES0_.exit
+  br i1 %28, label %_ZN4llvm12StringSwitchINS_5MachO12PlatformTypeES2_E4CaseENS_13StringLiteralES2_.exit105, label %_ZN4llvm12StringSwitchINS_5MachO12PlatformTypeES2_E4CaseENS_13StringLiteralES2_.exit97.thread
 
 _ZN4llvmeqENS_9StringRefES0_.exit.i.i94:          ; preds = %_ZN4llvmeqENS_9StringRefES0_.exit.i.i30
   %bcmp.i.i.i95 = call i32 @bcmp(ptr noundef nonnull dereferenceable(4) %14, ptr noundef nonnull dereferenceable(4) @.str.11, i64 4)
   %29 = icmp eq i32 %bcmp.i.i.i95, 0
-  br i1 %29, label %_ZN4llvm12StringSwitchINS_5MachO12PlatformTypeES2_E4CaseENS_13StringLiteralES2_.exit105, label %_ZNK4llvm9StringRef11starts_withES0_.exit
+  br i1 %29, label %_ZN4llvm12StringSwitchINS_5MachO12PlatformTypeES2_E4CaseENS_13StringLiteralES2_.exit105, label %_ZN4llvm12StringSwitchINS_5MachO12PlatformTypeES2_E4CaseENS_13StringLiteralES2_.exit97.thread
+
+_ZN4llvm12StringSwitchINS_5MachO12PlatformTypeES2_E4CaseENS_13StringLiteralES2_.exit97.thread: ; preds = %_ZN4llvmeqENS_9StringRefES0_.exit.i.i, %_ZN4llvmeqENS_9StringRefES0_.exit.i.i14, %_ZN4llvmeqENS_9StringRefES0_.exit.i.i94, %_ZN4llvmeqENS_9StringRefES0_.exit.i.i86
+  %.sroa.30.11.ph = phi i64 [ 4294967297, %_ZN4llvmeqENS_9StringRefES0_.exit.i.i14 ], [ 4294967296, %_ZN4llvmeqENS_9StringRefES0_.exit.i.i ], [ 0, %_ZN4llvmeqENS_9StringRefES0_.exit.i.i86 ], [ 0, %_ZN4llvmeqENS_9StringRefES0_.exit.i.i94 ]
+  br label %_ZN4llvm12StringSwitchINS_5MachO12PlatformTypeES2_E4CaseENS_13StringLiteralES2_.exit105
 
```

Comment:
Regression?

This change is introduced by different inputs to `jump-threading` (I am trying to figure out what happens).
And the input difference is introduced by SCCP folding a conditional br to uncond br, as follows:
<img width="1761" height="258" alt="image" src="https://github.com/user-attachments/assets/6c7e849c-0153-4770-888f-f8ff34cb4efa" />